vars/folioexecuteDataMigrationUtils.groovy

```diff
@@ -12,6 +12,19 @@ import org.folio.client.jira.JiraClient
 import org.folio.client.jira.model.JiraIssue
 import org.folio.karate.teams.TeamAssignment
 
+
+def getSlackColor(def buildStatus) {
+    if (buildStatus == 'STARTED') {
+        '#D4DADF'
+    } else if (buildStatus == 'SUCCESS') {
```

Comment:
case operator would look a bit more logical in this place.

Reply:
Agree, switch case better to use here
Or Map

Reply:
My dear friends kindly ask you to review it again